lib/mm_network.pm

```diff
@@ -74,7 +75,7 @@ sub configure_static_ip {
         my $mac = $net_conf->{fixed}->{mac};
 
         # Get default network adapter name
-        $device = script_output("grep $mac /sys/class/net/*/address |cut -d / -f 5") unless ($device);
+        $device //= script_output("grep $mac /sys/class/net/*/address |cut -d / -f 5");
```

Comment:
Hm, before the change this would have been equivalent to `|=`. If it is an empty string, `//=` wouldn't do anything. Not sure what is the right thing to do here.